examples/mnist/train_mnist.py

```diff
@@ -85,12 +85,14 @@ def main():
     trainer.extend(extensions.LogReport())
 
     # Save two plot images to the result dir
-    trainer.extend(
-        extensions.PlotReport(['main/loss', 'validation/main/loss'], 'epoch',
-                              file_name='loss.png'))
-    trainer.extend(
-        extensions.PlotReport(['main/accuracy', 'validation/main/accuracy'],
-                              'epoch', file_name='accuracy.png'))
+    if extensions.PlotReport.available:
```

Comment:
`extensions.PlotReport.available` is a function, thus this expression will be always evaluated as true. Is it reasonable to write this as a function-call form: `extensions.PlotReport.available()`?